Reconstruction/RecCalorimeter/options/runEcalReconstruction.py

```diff
@@ -0,0 +1,60 @@
+EVTMAX=100
+
+from Gaudi.Configuration import *
+
+from Configurables import ApplicationMgr, FCCDataSvc, PodioOutput
+
+podioevent   = FCCDataSvc("EventDataSvc")
+
+# reads HepMC text file and write the HepMC::GenEvent to the data service
+from Configurables import PodioInput
+podioinput = PodioInput("PodioReader", filename="/afs/cern.ch/user/n/novaj/public/FCCSW_tests/output_ecal.root", collections=["ECalHits", "ECalClusters", "GenParticles", "GenVertices"], OutputLevel=DEBUG)
```

Comment:
Sure, I forgot to ask where to put the file. Should I copy it to my FCC eos 
(/eos/fcc/users/n/novaj/) or somewhere else? I tried to run with a file at eos, everything runs fine until the end:
ApplicationMgr       INFO Application Manager Finalized successfully
ApplicationMgr       INFO Application Manager Terminated successfully
But the very last line is a bit worrying:
 *** Break *** segmentation violation
Any ideas what could have happened?